visual.js

```diff
@@ -81,7 +81,6 @@ function drawPiece(piece) {
         "B": "Bishop",
         "P": "Pawn"
     }
-    //logic to make which piece is necessary
```

Comment:
Why this is removed in this branch?

Reply:
my mistake. have added it back

Reply:
@austramarta please push the changes :)
Also - you can check changes like this (after committing) by `git diff HEAD~~..HEAD`